### DISCUSSION TITLE
Flashing a GeniusPro Problems

### DISCUSSION THREAD

**Discussion by jmfloyd** [dgtaheno/artillery-Genius#33] [Q&A]
Hello,

Thanks for this work.  I am running a Genius Pro, but am having troubles.

I get "no printer attached" all the time after flashing.  I could only run an update using STM32CubeProg, using the jumper, other methods did not work for me.  Now I have tried to revert it back to the stock artillery firmware with similar results. In all this I have varied the printer board rates from the TFT, powered to system up and down ...  with no success.

I had updated the TFT to the latest artillery version you had up earlier, and it was communicating with the original firmware that came with the printer (eg showing current bed and head temperatures. etc).  Now nothing.

Any suggestions?  Do you know if there is something special that needs to be set in the STM32CubeProg flashing (it is the one common activity in all my experiments)?  I havent been able to find any thing to help. 

It sounds like a flashing problem - there are no errors from the STM32 software.  I am running it on a linux system, and the DFU port comes up nicely, and is recognized.  

Cheers
John

**Comment by jmfloyd** [comment]
Extra info, I have tried to flash the motherboard using STM32CubeProg.  I get errors (?fatal?) that sector xxxx does not exist. xxxx is 0 to 5. Is this relevant.

After powering on after flashing, there is no usb serial connection available on the board's USB port.

**Reply by dgtaheno** [reply]
Hello @jmfloyd ,

Hope that you are doing well.

Did you tried changing the baudrate?

What I would you in your case would be:
Menu
![image](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/assets/62843933/d3fd81de-2061-4f1e-9648-dbcd497f2e57)
Settings
![image](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/assets/62843933/ba18c902-4c14-4432-b99f-4dea965613b3)
Connection 
![image](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/assets/62843933/e7cc6303-0b96-417b-8b96-be5225d9ea3f)
S.ports(please make sure that the connection is on in the lower left option, you can also try disconnect and connect here)
![image](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/assets/62843933/88863be4-61e5-4528-9fa6-d43e92f88190)
Select the printer
![image](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/assets/62843933/466fa32b-f9e3-4448-936c-9a49c4404aac)
If not 250000 baudrate, then select it. 
If 250000 selected, try with 115200.
![image](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/assets/62843933/62dd7bab-dbfa-44ab-b670-fafd5febe926)

Hope this will be a fix for you! Please let me know if not working so we can look for further solutions.

Best regards.

**Comment by dgtaheno** [comment]
Hi @jmfloyd ,

just saw that you closed the issue here https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/issues/34 . 

Glad to read that the issue was solved!

Hope that you can enjoy your printer now 👍🏻 

Best regards.
